### PR TITLE
Add Area and Distances info for AOIs

### LINKED ISSUE #661: Display bbox/polygons area and width/height
### Description of the current behavior

None

### Description of the desired behavior

These info should be displayed on
- [x] click on a aoi
- [x] while drawing
- [x] while editing

For polygons, display height/width of the enveloppe

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**
- Fix #661 

image

### CODE REVIEW

**Review by sebbousquet — APPROVED** [review]

File: src/app/components/map/aoi-dimensions/aoi-dimensions.component.scss [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    min-width: 215px;
+    background-color: white;
+    @include box-border();
+    padding: $sm-spacing;
+    .title_wrapper {
+        display: flex;
+        justify-content: space-between;
+        font-size: $default-font-size;
+        align-items: center;
+        margin-bottom: 5px;
```
Comment (QuCMGisaia) [inline src/app/components/map/aoi-dimensions/aoi-dimensions.component.scss:34]: Could be $sm-margin

File: src/app/components/map/aoi-dimensions/aoi-dimensions.component.scss [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+ @import '../../../../styles/app/sizes.scss';
+ @import '../../../../styles/app/mixin.scss';
+
+.wrapper {
+    border-radius: 4px;
```
Comment (QuCMGisaia) [inline src/app/components/map/aoi-dimensions/aoi-dimensions.component.scss:24]: Could be $default-border-radius

File: src/app/components/map/aoi-dimensions/aoi-dimensions.component.scss [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                .multiply {
+                    margin-right: $default-spacing;
+                    margin-left: $default-spacing;
+                }
+            }
+
+            .message {
+                color: rgb(143, 143, 143);
+                font-weight: 100;
+                font-size: 12px;
```
Comment (QuCMGisaia) [inline src/app/components/map/aoi-dimensions/aoi-dimensions.component.scss:76]: Could be $sm-font-size

File: src/assets/i18n/en.json [hunk 4]
```diff
@@ -86,5 +86,6 @@
 	"Double click to save this period": "Double click to save this period",
 	"Types TEXT, GEO_POINT are excluded": "Types 'TEXT', 'GEO_POINT' are excluded",
 	"Area Of Interest": "Area of interest",
-	"Fill": "Polygon"
+	"Fill": "Polygon",
+	"Current AOI": "Current AOI"
```
Comment (QuCMGisaia) [inline src/assets/i18n/en.json:90]: Not all keys from the fr.json are in the en.json. We should run an extract on the keys to translate before the final release.

**Review by QuCMGisaia — APPROVED** [review]

File: src/app/components/map/aoi-dimensions/aoi-dimensions.component.scss (hunk 2 above)
Comment (MohamedHamouGisaia) [reply src/app/components/map/aoi-dimensions/aoi-dimensions.component.scss:24]: fixed

File: src/app/components/map/aoi-dimensions/aoi-dimensions.component.scss (hunk 3 above)
Comment (MohamedHamouGisaia) [reply src/app/components/map/aoi-dimensions/aoi-dimensions.component.scss:76]: fixed

File: src/assets/i18n/en.json (hunk 4 above)
Comment (MohamedHamouGisaia) [reply src/assets/i18n/en.json:90]: yes I agree

**Pull request merged**